client/settings/deposits/index.js

```diff
@@ -72,6 +75,17 @@ const monthlyAnchors = [
 ];
 
 const CustomizeDepositSchedule = () => {
+	const {
+		accountStatus: {
+			deposits: { minimum_deposit_amounts: minimumDepositAmounts },
+		},
+		storeCurrencies,
+	} = useContext( WCPaySettingsContext );
```

Comment:
If for some reason (maybe unexpected future change from server), `accountStatus.deposits` isn't defined, this line will throw an error, right?

Maybe we can be more defensive here?

Reply:
I don't think we can be too defensive about the server not returning the expected response. If we need to eventually move/change the payload then it should be done in a backwards-compatible way.